custom/icds_reports/tests/agg_tests/reports/test_demographics.py

```diff
@@ -165,317 +165,3 @@ def test_data_monthly_adolescent_girls(self):
             "help_text": percent_adolescent_girls_enrolled_help_text()
         }
         self.assertDictEqual(expected, data['records'][2][1])
-
```

Comment:
The test case were written for testing daily data cases and since we are pulling from monthly table only now I removed the test cases.

Do let me know if you think it should be handled differently.